NotificationHubs/src/com/windowsazure/messaging/NotificationHub.java

```diff
@@ -999,7 +997,7 @@ public void sendDirectNotificationAsync(Notification notification, List<String>
             throw new RuntimeException(e);
         }
 
-        post.setBody(baoStream.toByteArray(), ContentType.MULTIPART_MIXED);
+        post.setBody(baoStream.toByteArray(), ContentType.parse(entity.getContentType()));
```

Comment:
Any unit tests we can add to prevent a regression here?

Reply:
Added unit test